test/dynamo/test_export_mutations.py

```diff
@@ -57,18 +44,6 @@ def forward(self, x):
 
         self.check_failure_on_export(Foo(), torch.randn(3, 2))
 
-    def test_module_attribute_mutation_violation_positive_4(self):
```

Comment:
So this works now? Just gets inlined into a constant on the RHS, is that why? 

But I don't see `self.a` being initialized, so isn't that a new attribute, like `test_module_attribute_mutation_violation_positive_3`? Confused about the mental model here.

Reply:
Yeah, self.a is a new attribute, but it's a constant like in `mutation_violation_positive_3`. `mutation_violation_positive_3` is failing due to a different error, so I moved it to `test_export` instead.